NAMESPACE

```diff
@@ -48,7 +48,49 @@ export(filter_by_file_extension)
 export(filter_by_filepath_substring)
 export(filter_by_last_files_change)
 export(format_name_email)
+export(get_bugzilla_project_key)
+export(get_code_language)
+export(get_cveid_regex)
 export(get_date_from_commit_hash)
+export(get_dv8_flaws_params)
+export(get_dv8_folder_path)
+export(get_enumeration_commits)
+export(get_file_extensions)
+export(get_filter_commit_size)
+export(get_git_branches)
+export(get_git_repo_path)
+export(get_github_commit_path)
+export(get_github_issue_or_pr_comment_path)
+export(get_github_issue_path)
+export(get_github_issue_search_path)
+export(get_github_keys)
+export(get_github_owner)
+export(get_github_pull_request_path)
+export(get_github_repo)
+export(get_issue_id_regex)
+export(get_jira_domain)
+export(get_jira_issues_comments_path)
+export(get_jira_issues_path)
+export(get_jira_keys)
+export(get_jira_project_key_name)
+export(get_keep_dependencies_type)
+export(get_mbox_archive_type)
+export(get_mbox_domain)
+export(get_mbox_key_indexes)
+export(get_mbox_mailing_list)
+export(get_mbox_path)
+export(get_nvdfeed_folder_path)
+export(get_parsed)
```

Comment:
This function name is too cryptic and it is not a get, it is a parse. So it should be named instead parse_project_config().

Reply:
I use the `get_parsed()` function to parse the `tools.yml` file too.  For example, one of the notebooks configuration section would contain: 
`tool <- get_parsed("tools.yml")`
`conf <- get_parsed("conf/geronimo.yml")`

Would renaming `get_parsed()` to `parse_project_config()` and using this renamed parser function to parse the `tools.yml` and project configuration files still be accurate as a name? If not, `parse_config()` sounds more accurate.